apps/wallet-mobile/src/TxHistory/TxHistoryList/TxHistoryList.tsx

```diff
@@ -48,6 +48,7 @@ export const TxHistoryList = (props: Props) => {
         ItemSeparatorComponent={() => <Spacer height={16} />}
         renderSectionHeader={({section: {data}}) => <DayHeader ts={data[0].submittedAt} />}
         sections={groupedTransactions}
+        renderSectionFooter={() => <Spacer height={12} />}
```

Comment:
added the spacer here in thee footer, so the 1st row doesn't bloat blank space on screen